workers/advisor/src/main/kotlin/advisor/AdvisorWorker.kt

```diff
@@ -20,10 +20,9 @@
 package org.eclipse.apoapsis.ortserver.workers.advisor
 
 import org.eclipse.apoapsis.ortserver.dao.dbQuery
-import org.eclipse.apoapsis.ortserver.model.AdvisorJob
-import org.eclipse.apoapsis.ortserver.model.JobStatus
 import org.eclipse.apoapsis.ortserver.workers.common.JobIgnoredException
 import org.eclipse.apoapsis.ortserver.workers.common.OrtRunService
+import org.eclipse.apoapsis.ortserver.workers.common.OrtRunService.Companion.validateForProcessing
```

Comment:
Commit message nit: "Use the *new* validation function..."

Reply:
Rewritten this commit message as well.